nidaqmx/system/system.py

```diff
@@ -34,13 +37,24 @@ class System(object):
     operations on DAQ hardware, and creates classes from which you can get
     information about the hardware.
     """
+    def __init__(self, interpreter = None):
+        self._interpreter = interpreter
 
     @staticmethod
     def local():
         """
         nidaqmx.system.system.System: Represents the local DAQmx system.
         """
-        return System()
+        interpreter = _library_interpreter.LibraryInterpreter(encoding='windows-1251')
```

Comment:
Where did you copy windows-1251 from? windows-1251 is the encoding for languages like Russian and Ukrainian that use Cyrillic script. windows-1252 is Latin 1, which is the encoding for English, French, and German (among others).

`nicaiu.dll` uses ANSI/MBCS (windows-1252, windows-932, etc.). Both `nicai_utf8.dll` and gRPC use UTF-8.

We should avoid hardcoding a specific encoding, if possible. However, modern Linux systems use UTF-8 by default, so if we are using `nicaiu.dll`, it may be necessary to explicitly specify a MBCS encoding for the library interpreter.

Reply:
Sure Brad, thanks for pointing that out, i reused the implementation from nimi python and this value is from there.

Reply:
There is an open issue about that: https://github.com/ni/nimi-python/issues/1820